pkg/retry/retry.go

```diff
@@ -9,4 +9,6 @@ const (
 	Interval = 15 * time.Second
 	// Timeout is the total time we will wait for an event to occur.
 	Timeout = time.Minute * 10
+	// ResourceChangeTimeout is the total time waited for a change (create/update/delete) to take place
+	ResourceChangeTimeout = time.Minute * 2
```

Comment:
I would prefer this change be done in a separate PR.

Reply:
Change is directly related to this PR as discussed here
https://github.com/openshift/windows-machine-config-operator/pull/618/commits/2998aa57b35da5f8380458269c635f245f920c38#r698607183

Reply:
Not sure why the link doesn't work so here's the conversation:

 aravindhp on Aug 30
> You are introducing a behavior change here. Why are you doing that?

 saifshaikh48 on Aug 31
> I was seeing a flake here as setting the Upgradeable condition takes ~20secs itself so the previous timeout was too low. Also for consistency. Updated the commit message to make it more clear.

Reply:
OK, please call out in the commit message that the reason for increasing it was Upgradeable condition taking longer to be updated.